scholia/app/templates/venue.html

```diff
@@ -360,6 +394,11 @@ <h3 id="Venues cited from">Venues cited from</h3>
 <table class="table table-hover" id="citing-venues"></table>
 
 
+<h2>Retracted articles</h2>
```

Comment:
There is a discrepancy between the Javascript variable, HTML id and headline. I wonder if we can align them?

Reply:
Do you mean, giving them all the same name e.g. Retracted_articles for all variables?